### PR TITLE
Issue template for bug report.

### PR DESCRIPTION
**Pull request by laxmikantbpandhare**
As discussed in today's community meeting, added issue template for bug report.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+<!-- A clear and concise description of what you expected to happen (or insert a code snippet). -->
+
+#### Environment
+
+**Kubernetes cluster type:**
+
+<!-- The type of cluster used for testing/deployment, ex. "vanilla", "OpenShift" -->
+
+`$ java-operator-sdk version`
```
Comment (metacosm) [inline .github/ISSUE_TEMPLATE/bug-report.md:37]: This implies running a CLI which doesn't exist…

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+<!-- Insert the output of `operator-sdk version` here. -->
+
+`$ java version`
```
Comment (metacosm) [inline .github/ISSUE_TEMPLATE/bug-report.md:41]:
```suggestion
`$ java -version`
```

**Review by metacosm — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (laxmikantbpandhare) [reply .github/ISSUE_TEMPLATE/bug-report.md:37]: So, how one can check about the version he is using. `pom.xml` file?

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 2 above)
Comment (laxmikantbpandhare) [reply .github/ISSUE_TEMPLATE/bug-report.md:41]: Modified.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (laxmikantbpandhare) [reply .github/ISSUE_TEMPLATE/bug-report.md]: @metacosm  - Modified the instructions. Asked user to verify `java-operator-sdk` version from `pom.xml` file.

**Review by metacosm — DISMISSED** [review]

**Review by metacosm — CHANGES_REQUESTED** [review]
Please rebase your PR so that all commits follow the conventional commit message format: https://www.conventionalcommits.org/en/v1.0.0/

**Comment by laxmikantbpandhare** [comment]
> Please rebase your PR so that all commits follow the conventional commit message format: https://www.conventionalcommits.org/en/v1.0.0/

@metacosm I updated all the messages with `style` tag as we are adding template/styling for our issue/bug creation process. I squashed all the commits as well.

**Review by metacosm — APPROVED** [review]

**Comment by metacosm** [comment]
Thank you!

**Pull request merged**